### ISSUE TITLE
Ticketform snippet : section is empty

### ISSUE THREAD

**Issue by squarou** [bezumkin/Tickets#140]
Hi !
I'm just trying to use tickets wihich seems so great !

But I can't make TicketForm working. The section select remains empty. I've created 1 section (1540) but doesn't appears.
Here's my call of snippet :
`[[!TicketForm?
	&parents=`1540`
]]`
Doesn't work neither with the same result :
`[[!TicketForm]]`

I'm I missing something ?

**Comment by bezumkin** [comment]
Your logged user must be in a group with **TicketUserPolicy** enabled
<img width="1033" alt="2016-05-25 19 56 17" src="https://cloud.githubusercontent.com/assets/1257284/15540667/d885ab7a-22b2-11e6-94f8-125a46baa72c.png">


And parents - it is for parents of categories. E.g, for parent of resource 1540.

**Comment by squarou** [comment]
Thanks. That works !
Notice that if the section is part of a resource group, it doesn't work.
For example, I've got 2 groups of users : member and CA_member
I don't want member to see the section. So I use the resource group tab and check the resource group "CA_frontend". 
I had to go to Resource Group Access
![screenshot-2016-05-25 23-34-25](https://cloud.githubusercontent.com/assets/6338874/15557247/4d8fe92a-22d2-11e6-8630-9fe9f6c77a56.png)
 and add TicketSectionPolicy.
Hope it can help others who use Tickets and resource groups.
Thanks a lot !
S.
